lib/Yield/PTML.cpp

```diff
@@ -535,19 +543,71 @@ static std::string taggedText(const PTMLBuilder &ThePTMLBuilder,
   return Result;
 }
 
+static std::string instructionPrefix(const PTMLBuilder &ThePTMLBuilder,
+                                     const yield::Instruction &Instruction) {
+  std::string Result;
+
+  if (options::InstructionAddressColumnWidth) {
+    std::string SerializedAddress = Instruction.Address().toString();
```

Comment:
I'd just dump the address, not the whole `MetaAddress`. Too much noise for little return.

A basic block can only be of one type (can't mix Thumb and regular ARM in a single block), but even a function mixing ARM and Thumb is rare enough that I don't think it's worth all the noise.

Reply:
Done.